pkg/index/job/correction/service/corrector.go

```diff
@@ -600,6 +614,7 @@ func (c *correct) loadInfos(ctx context.Context) (err error) {
 		return true
 	})
 	infoMap.Range(func(addr string, info *payload.Info_Index_Count) bool {
+		log.Infof("index info: addr(%s), stored(%d), uncommitted(%d)", addr, info.GetStored(), info.GetUncommitted())
```

Comment:
> **NOTE**
Logs are also output at the same time so that there is no need loop for debugging on the caller side.